src/generators/default/string/index.ts

```diff
@@ -104,3 +104,10 @@ export const Cuid2Generator = Generator({
 		core.utils.filter.checks(def.checks, 'cuid2') !== undefined,
 	output: ({ def, core }) => formatString(core, def, core.utils.random.cuid2()),
 });
+
+export const DateTimeGenerator = Generator({
+	schema: ZodString,
+	filter: ({ def, core }) =>
+		core.utils.filter.checks(def.checks, 'datetime') !== undefined,
```

Comment:
What about using a `hasCheck`/`includesCheck`/... helper that just returns a boolean?

Reply:
I like that idea. Maybe refactor so it's:

```ts
core.utils.checks.find( def.checks, 'type' ) // instead of filter since we're only returning one value
core.utils.checks.has( def.checks, 'type' ) // boolean result
```

Reply:
Take a look at this PR (https://github.com/timdeschryver/zod-fixture/pull/31).
I think it cleans things up nicely.